filament/src/PostProcessManager.h

```diff
@@ -144,6 +143,10 @@ class PostProcessManager {
     FrameGraphId<FrameGraphTexture> resolve(FrameGraph& fg,
             const char* outputBufferName, FrameGraphId<FrameGraphTexture> input) noexcept;
 
+    FrameGraphId<FrameGraphTexture> resolveNoCheck(FrameGraph& fg,
```

Comment:
Add a comment describing what this new method does.. and yeah, I think I prefer `resolveBaseLevel`. At least, I can't think of anything better.